requirements-build.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
+pyinstaller==6.6.0
```

Comment:
Instead of doing it with these extra files, we should do optional targets in the pyproject.toml

```
[project.optional-dependencies]
build = [
  'pyinstaller==6.6.0',
]
tests = [
  'pytest-cov',
]
```